### ISSUE TITLE
Total Commander progress bug

### ISSUE THREAD

**Issue by sexzyura** [dremin/RetroBar#686]
found a bug with the Retrobar and Total Commander program. when the file copying process starts in the totalcmd, it is indicated progress accordingly on the program icon on the task bar. the bug consists in the fact that after performing the copying process, this inscription remains on the panel [0%]
![issue2](https://github.com/dremin/RetroBar/assets/141490640/df9d3000-41ee-4d29-a38e-3efc2d150529)

**Comment by xoascf** [comment]
That is a bug in Total Commander as it fails to return to `TBPF_NOPROGRESS` upon completion of copying.

More info: https://learn.microsoft.com/en-us/windows/win32/api/shobjidl_core/nf-shobjidl_core-itaskbarlist3-setprogressstate

**Comment by sexzyura** [comment]
ghisler said that this is a 
![Screenshot 2023-09-19 at 18-20-27 progress bar issue - Total Commander](https://github.com/dremin/RetroBar/assets/141490640/472333ca-2a7d-43ac-b980-d6d36cfc7050)
retrobar bug

**Comment by ssp88** [comment]
![Screenshot 2023-10-21 at 13-25-55 progress bar issue - Total Commander](https://github.com/dremin/RetroBar/assets/141490640/64b9d6c5-e9ac-4adb-aa6d-47fd91408819)

**Comment by dremin** [comment]
While it is true that Total Commander is setting the `TBPF_NOPROGRESS` state, it then immediately after sets a `TBPF_NORMAL` state, which causes `0%` to appear in RetroBar. We can see this in the last two lines of the RetroBar debug logs:

```
[10/22/2023 6:58:53 PM] Debug: TasksService: ITaskbarList: SetProgressState HWND:658880 Flags: 2
[10/22/2023 6:58:53 PM] Debug: TasksService: ITaskbarList: SetProgressValue HWND:658880 Progress: 8520
[10/22/2023 6:58:54 PM] Debug: TasksService: ITaskbarList: SetProgressState HWND:658880 Flags: 2
[10/22/2023 6:58:54 PM] Debug: TasksService: ITaskbarList: SetProgressValue HWND:658880 Progress: 20971
[10/22/2023 6:58:54 PM] Debug: TasksService: ITaskbarList: SetProgressState HWND:658880 Flags: 2
[10/22/2023 6:58:54 PM] Debug: TasksService: ITaskbarList: SetProgressValue HWND:658880 Progress: 28179
[10/22/2023 6:58:54 PM] Debug: TasksService: ITaskbarList: SetProgressState HWND:658880 Flags: 2
[10/22/2023 6:58:54 PM] Debug: TasksService: ITaskbarList: SetProgressValue HWND:658880 Progress: 37354
[10/22/2023 6:58:54 PM] Debug: TasksService: ITaskbarList: SetProgressState HWND:658880 Flags: 2
[10/22/2023 6:58:54 PM] Debug: TasksService: ITaskbarList: SetProgressValue HWND:658880 Progress: 45873
[10/22/2023 6:58:54 PM] Debug: TasksService: ITaskbarList: SetProgressState HWND:658880 Flags: 2
[10/22/2023 6:58:54 PM] Debug: TasksService: ITaskbarList: SetProgressValue HWND:658880 Progress: 52426
[10/22/2023 6:58:55 PM] Debug: TasksService: ITaskbarList: SetProgressState HWND:658880 Flags: 2
[10/22/2023 6:58:55 PM] Debug: TasksService: ITaskbarList: SetProgressValue HWND:658880 Progress: 55703
[10/22/2023 6:58:55 PM] Debug: TasksService: ITaskbarList: SetProgressState HWND:658880 Flags: 2
[10/22/2023 6:58:55 PM] Debug: TasksService: ITaskbarList: SetProgressValue HWND:658880 Progress: 64222
[10/22/2023 6:58:55 PM] Debug: TasksService: ITaskbarList: SetProgressState HWND:658880 Flags: 0
[10/22/2023 6:58:55 PM] Debug: TasksService: ITaskbarList: SetProgressState HWND:658880 Flags: 2
```

**Comment by homandr** [comment]
This issue is fixed in TC 11.50. See https://www.ghisler.ch/board/viewtopic.php?p=461133.
From the change log:
`11.04.24 Fixed: Set taskbar progress state to TBPF_NOPROGRESS instead of TBPF_NORMAL after file operations (32/64)`
